Podcast/FetchUserFollowersByIDRequest.swift

```diff
@@ -45,6 +45,9 @@ class FetchUserFollowsByIDRequest: EndpointRequest {
             let userJSON = followJSON.1[type.userKey]
             print(userJSON)
             let user = User(json: userJSON)
+            if type == .Followings {
+                user.isFollowing = true
```

Comment:
Don't they set this for us? What happens if we're following someone who follows us, they must send us that.

Reply:
they do not send it to us! It's extra fields to send over a network when we don't need it because inherently if we are getting the followings we know we are following them!

Reply:
But we definitely need it for followers though right?

Reply:
Yes, it wasn't working before I put it in.